.gitignore

```diff
@@ -11,7 +11,5 @@
 /vendor
 environment.sh
 docker.env
-/.idea/
-/.idea/vcs.xml
-docker.env
```

Comment:
Do we want to remove this from the gitignore? This could contain secrets which could be accidently pushed to github?

Reply:
they're duplicates - they're still there on docker.env on line 13 and `.idea` on line 14. the leading `/` does nothing and it ignores the entire directory and all its contents if it's a dir